src/datetime.js

```diff
@@ -1114,7 +1114,7 @@ export default class DateTime {
    * @type {number}
    */
   get offset() {
-    return this.isValid ? this.zone.offset(this.ts) : NaN;
```

Comment:
Why does this need a `+`? Seems like it would already be a number?

Reply:
but i can be null as well if in some crazy case the invalid is false...